cost/azure/instances_log_analitics_utilization/README.md

```diff
@@ -0,0 +1,48 @@
+## Azure Instances Utilization from Log Analytics
+
+### What it does
+
+This Policy Template uses monitoring metrics from Log Analytics for the last 30 days to identify underutilized instances. This is meant to be run as a monthly policy. 
```

Comment:
Monthly is not an option in the frequency.

Reply:
Revised to weekly until monthly is available.